src/util/fp.js

```diff
@@ -4,9 +4,11 @@
  * - Add polyfills.
  */
 
-import { curry } from 'lodash/fp';
+import { flow, curry } from 'lodash/fp';
```

Comment:
Why do we have our own `fp` module while using `lodash/fp`?

Reply:
I didn't get to writing our own `flow` and `curry` implementations, yet. Once we do that we can drop the dependency. I think, in particular with these to functions, tree shaking should actually work. So we are really just adding them to the bundle not all the rest of lodash.